src/fonts/common_metrics.ts

```diff
@@ -1,82 +0,0 @@
-export const CommonMetrics = {
```

Comment:
So this file has been replaced by what is now in tables.ts? Obviously the new CommonMetrics handles font information as well....

I think in a future PR, we should move that structure out into its own file. 

It can be called `Metrics` and stored in `metrics.ts`.

Reply:
Agree